yt_dlp/extractor/amazon.py

```diff
@@ -20,28 +32,32 @@ class AmazonStoreIE(InfoExtractor):
                 'thumbnail': r're:^https?://.*\.jpg$',
                 'duration': 34,
             },
-        }]
+        }],
+        'expected_warnings': ['Unable to extract data'],
     }, {
         'url': 'https://www.amazon.in/Sony-WH-1000XM4-Cancelling-Headphones-Bluetooth/dp/B0863TXGM3',
         'info_dict': {
             'id': 'B0863TXGM3',
-            'title': 'md5:d1d3352428f8f015706c84b31e132169',
+            'title': 'md5:f62b7bc5591a6107553bfff00d6e49fc',
```

Comment:
If these are not static, use weaker test definition. Eg:
```suggestion
            'title': str,
```